sem_covid/entrypoints/etl_dags/eu_cellar_covid.py

```diff
@@ -417,52 +421,70 @@
 GROUP BY ?work ?title
 ORDER BY ?work ?title"""
 
-sources = {
-    "EurLex": {"json": config.EU_CELLAR_JSON, "query": EURLEX_QUERY},
-    "Extended EurLex part 1": {"json": config.EU_CELLAR_EXTENDED_JSON, "query": EURLEX_EXTENDED_QUERY},
-}
-
-
-def make_request(query, wrapperSPARQL):
-    wrapperSPARQL.setQuery(query)
-    wrapperSPARQL.setReturnFormat(JSON)
-
-    return wrapperSPARQL.query().convert()
-
-
-def get_single_item(query, json_file_name, wrapperSPARQL, minio, transformer):
-    response = make_request(query, wrapperSPARQL)['results']['bindings']
-    eurlex_json_dataset = transformer(response)
-    uploaded_bytes = minio.put_object(json_file_name, json.dumps(eurlex_json_dataset).encode('utf-8'))
-    logger.info(f'Save query result to {json_file_name}')
-    logger.info('Uploaded ' + str(
-        uploaded_bytes) + ' bytes to bucket [' + config.EU_CELLAR_BUCKET_NAME + '] at ' + config.MINIO_URL)
-    return eurlex_json_dataset
+EU_CELLAR_CORE_KEY = "eu_cellar_core"
+EU_CELLAR_EXTENDED_KEY = "eu_cellar_extended"
+
+
+def unify_dataframes_and_mark_source(list_of_data_frames: List[pd.DataFrame], list_of_flags: List[str],
```

Comment:
in the next round of refactorings, we will need to move all the helper functions into a common module, and reuse them across the rest of the DAGs.